src/main/scala/mesosphere/marathon/event/http/HttpEventStreamHandleActor.scala

```diff
@@ -76,11 +76,11 @@ class HttpEventStreamHandleActor(
       //Do not act any longer on any event.
       context.become(Actor.emptyBehavior)
     case _ =>
-      log.warning(s"Could not send message to $handle", ex)
+      log.warning("Could not send message to {} reason: {}", handle, ex)
   }
 
   private[this] def dropEvent(event: MarathonEvent): Unit = {
-    log.warning(s"Ignore event $event for handle $handle (slow consumer)")
+    log.warning("Ignore event {} for handle {} (slow consumer)", event, handle)
```

Comment:
Looked like the default usage in our code base. 
My preference would be to use a clever scala logging system, where we don't have to decide which way to use...